packages/wdio-cli/src/constants.js

```diff
@@ -1,7 +1,8 @@
-import { version } from '../package.json'
 import { validateServiceAnswers, hasFile } from './utils'
 
-export const CLI_EPILOGUE = `Documentation: https://webdriver.io\n@wdio/cli (v${version})`
+const pkg = require('../package.json')
```

Comment:
Import wouldn't work here?

Reply:
nope, TS error saying that `package.json` is not part of the source files.

Reply:
can we overcome it by allowing resolution of json modules? `resolveJsonModule`

Reply:
the problem is that TS expects all files you import to be within the files you define in `src`. I don't want to set it to `src: "."` because then we get a folder structure like this:

```
.
└── my-project/
    ├── node_modules/
    │   ├── ...
    │   └── lodash.clonedeep/
    │       ├── package.json
    │       └── index.js
    └── packages/
        └── .../
            └── my-sub-package/
                ├── build/
                │   ├── package.json
                │   └── src/
                │       └── file.js
                ├── src/
                │   └── file.ts
                ├── package.json
                └── ...
```
note the double package.jsons which also causes other TS issues.